test/tests/functional/pbs_snapshot_unittest.py

```diff
@@ -257,7 +257,15 @@ def test_capture_server(self):
                     # Find the common paths between 'server' & the file
                     common_path = os.path.commonprefix([file_fullpath,
                                                         svr_fullpath])
-                    self.assertEqual(os.path.basename(common_path), "server")
+                    try:
+                        self.assertEqual(os.path.basename(common_path),
+                                         "server")
+                    except AssertionError:
+                        # Check if this was a core file, which would
+                        # explain why it was captured
+                        if CORE_DIR in file_fullpath:
```

Comment:
As part of capture_server(), will we also capture core files in another directory apart from the server ones?

Reply:
No we don't, we only capture server's core files. The thing is, we store them in a separate path inside the snapshot: <snapshot name>/core_file_bt
The test is checking that the only data captured was inside directories that shared a common path with <snapshot name>/server, which 'core_file_bt' will not. So, that's why I made the exception.

Did you want me to check that CORE_DIR didn't capture core files from other daemons?

Reply:
Yes. I think we should do that. It should be simple as we are creating a different directory for every daemon. If we have only the server_priv directory in core_file_bt/ that should be good enough check.